app/src/main/java/org/oppia/app/onboarding/OnboardingFragmentPresenter.kt

```diff
@@ -55,10 +56,20 @@ class OnboardingFragmentPresenter @Inject constructor(
       override fun onPageSelected(position: Int) {
         getOnboardingViewModel().slideChanged(ViewPagerSlide.getSlideForPosition(position))
         selectDot(position)
+        onboardingStatusBarColorUpdate(position)
       }
     })
   }
 
+  private fun onboardingStatusBarColorUpdate(position: Int) {
+    when (position) {
+      0 -> StatusBarColor.statusBarColorUpdate(R.color.onboarding_1_status_bar, activity, false)
+      1 -> StatusBarColor.statusBarColorUpdate(R.color.onboarding_2_status_bar, activity, false)
+      2 -> StatusBarColor.statusBarColorUpdate(R.color.onboarding_3_status_bar, activity, false)
+      3 -> StatusBarColor.statusBarColorUpdate(R.color.onboarding_4_status_bar, activity, false)
```

Comment:
Add an else clause also, which can point to `#003933` as default.

Reply:
+1

Reply:
Done